build/build.proj

```diff
@@ -88,7 +80,7 @@
 
   </Target>
 
-  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;BuildModernVsixPackages;Test" />
-  <Target Name="Rebuild" DependsOnTargets="RestorePackages;RebuildSolution;RebuildModernVsixPackages;Test" />
+  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;Test" />
```

Comment:
But this means that Jenkins won't build the modern vsixes right? can we instead make this conditionally exclude? i.e Microsoft can call build.cmd /excludemodernvsix and build.cmd /modernvsixesonly but build.cmd builds all?